lib/utils.js

```diff
@@ -22,6 +22,37 @@ export function calculateSlippageAmount(value) {
   return value.mul(SLIPPAGE_PERCENTAGE).div(100)
 }
 
+export function useCheckEmailForAddress() {
+  const [userExists, setUserExists] = useState(false)
```

Comment:
It seems some part of the implementation is missing? `userExists` is not returned, and `setUserExists()` is not called.

Reply:
Yeah, this part is not finished as I'm testing the API.